src/app/ubs/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.ts

```diff
@@ -353,6 +353,23 @@ export class UbsAdminTableComponent implements OnInit, AfterViewChecked, OnDestr
     return `${this.selection.isSelected(row) ? 'deselect' : 'select'} row ${row.id + 1}`;
   }
 
+  isStatus(row): string {
+    switch (row.orderStatus) {
+      case OrderStatus.CANCELED:
```

Comment:
you can group cases

case OrderStatus.CANCELED:
case OrderStatus.BROUGHT_IT_HIMSELF:
        return '#CCCCCC';

Reply:
thanks, added this change